assets/src/stories-editor/components/with-meta-block-edit.js

```diff
@@ -38,18 +38,23 @@ class MetaBlockEdit extends Component {
 			width,
 		} = attributes;
 
-		// If not selected, only proceed if height or width has changed.
-		if (
-			! isSelected &&
-			prevProps.attributes.height === height &&
-			prevProps.attributes.width === width &&
-			isEqual( prevProps.fontSize, fontSize )
-		) {
-			return;
+		// If not selected, only change font size if height or width has changed.
+		const checkFontSize = (
+			isSelected ||
+			(
+				! isSelected &&
```

Comment:
`! isSelected ` may not be needed, as it will only get to this point if it's `false` or 'falsey'

Reply:
Of course, this is a very minor point.